src/networking/server/plugin.rs

```diff
@@ -0,0 +1,112 @@
+use std::{net::SocketAddr, sync::Arc};
+
+use bevy::prelude::{AppBuilder, Plugin};
+use quinn::{crypto::rustls::TlsSession, generic::Incoming};
+
+use bevy::prelude::IntoQuerySystem;
+use futures::StreamExt;
+
+use tokio::sync::mpsc::{unbounded_channel, UnboundedSender};
+use tracing::info;
+
+use crate::networking::{
+    events::{
+        ReceiveEvent,
+        SendEvent
+    },
+    systems::{
+        NetworkConnections,
+        SessionEventListenerState,
+        handle_connection,
+        receive_net_events,
+        send_net_events
+    }
+};
+
+// Add this plugin to start a server which sends and receives packets to a large number of network connections
```

Comment:
```suggestion
/// Add this plugin to start a server which sends and receives packets to a large number of network connections
```
Turn comment to doc comment